lib/agent.js

```diff
@@ -257,3 +257,52 @@ Agent.prototype.handleUncaughtExceptions = function (cb) {
 Agent.prototype.flush = function (cb) {
   this._instrumentation.flush(cb)
 }
+
+Agent.prototype.lambda = function wrapLambda (name, fn) {
+  var agent = this
+  return function wrappedLambda (payload, context, callback) {
+    var trans = agent.startTransaction()
+    if (!trans) {
+      return fn(payload, context, callback)
+    }
+
+    trans.type = 'lambda'
+    trans.name = name
+
+    function wrapLambdaCallback (callback) {
```

Comment:
Yeah, I can do that. I just left it where it was because it originally was part of the call later, but I pulled it out to add support for the context functions later.

Reply:
fixed